packages/deploy-tooling/test/unit/cli/index.test.ts

```diff
@@ -2,20 +2,25 @@ import type { ParseOptions } from 'commander';
 import { join } from 'path';
 import { createCommand, runDeploy, runUndeploy } from '../../../src/cli';
 import * as cliArchive from '../../../src/cli/archive';
-import { mockedUi5RepoService } from '../../__mocks__';
+import { mockedUi5RepoService, mockedProvider } from '../../__mocks__';
 import { Command } from 'commander';
 import fs from 'fs';
 import { ToolsLogger } from '@sap-ux/logger';
 
 describe('cli', () => {
     const fixture = join(__dirname, '../../test-input/');
     const target = 'https://target.example';
+    const env = process.env;
```

Comment:
Most likely no problem but I sometimes have weird behaviors with assignments like that if I don't put them into `beforeAll`. Could you please move it there, it would then also match the corresponding call in `afterAll`?

Reply:
Done.